Core/Command.cs

```diff
@@ -2568,7 +2570,9 @@ private void ShowHelp(string sFile = "index.txt")
                     objFile.Close();
                 }
             }
+            #pragma warning disable CS0168
```

Comment:
Can we make a comment as to why we are disabling this error? If it does not apply in this case great, but if we are doing it just to get rid of a lint warning I would push back on it.